src/index.js

```diff
@@ -13,4 +10,7 @@ function mobileMenu() {
 function closeMenu() {
   hamburger.classList.remove('active');
   navMenu.classList.remove('active');
-}
\ No newline at end of file
+}
+
+hamburger.addEventListener('click', mobileMenu);
+navLink.forEach((n) => n.addEventListener('click', closeMenu));
```

Comment:
Great job on defining the function before using it.
However, you need also to show up all the menus whenever your user click or tap on your mobile menu, as presented in the following screenshot
![preview](https://user-images.githubusercontent.com/47100064/124231781-6d56c300-db19-11eb-9fd7-05597a33daa9.png)